spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'User Dashboard Page' do
+  before(:each) do
+    @user1 = User.create!(name: 'Chad', email: 'chad1@gmail.com')
+    @user2 = User.create!(name: 'Jessica', email: 'jessica2@gmail.com')
+    @user3 = User.create!(name: 'Fiona', email: 'Fiona3@gmail.com')
+  end
+
```

Comment:
Can we add the following test to ensure that the home button and Viewing Party title show up on every page?
*If you know of a more efficient way to ensure something is showing up on every page, please let me know!"
*I added a global partial to render the Home button and Viewing Party title to every new page that is created on another branch, so the test should pass after we get everything merged (it's ok to put it in as failing right now)

it 'should contain title of Viewing Party and a Home button that will redirect the user
    the landing page' do
    visit root_path
    within('#dashboard') do
      expect(page).to have_content('Viewing Party')
      expect(page).to have_link('Home')
      click_link('Home')
      expect(current_path).to eq(root_path)
    end
  end

Reply:
I added the test, it is currently failing but we can resolve that when it's merged.